SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp

```diff
@@ -700,58 +701,98 @@ void CDiscAdjMultizoneDriver::DirectIteration(unsigned short iZone, RECORDING ki
                                            solver_container, numerics_container, config_container,
                                            surface_movement, grid_movement, FFDBox, iZone, INST_0);
 
+}
+
+void CDiscAdjMultizoneDriver::Print_DirectResidual(RECORDING kind_recording) {
+
+  /*--- Helper lambda func to return lenghty [iVar][iZone] string.  ---*/
+  auto iVar_iZone2string = [](unsigned short ivar, unsigned short izone) {
+    return "[" + std::to_string(ivar) + "][" + std::to_string(izone) + "]";
+  };
+
   /*--- Print residuals in the first iteration ---*/
 
   if (rank == MASTER_NODE && kind_recording == RECORDING::SOLUTION_VARIABLES) {
 
-    auto solvers = solver_container[iZone][INST_0][MESH_0];
+    const unsigned short fieldWidth = 15;
```

Comment:
Unfortunately I had to increase to a min of 14 to fit e.g. `rms_Flow[0][0]` into the table. The fixed screen output size for the primal is currently 12... that is not ideal but better than before imo